ui/lib/pki/addon/components/pki-sign-intermediate-form.hbs

```diff
@@ -3,12 +3,12 @@
   <Toolbar />
   <main data-test-sign-intermediate-result>
     <div class="box is-sideless is-fullwidth is-shadowless">
-      <AlertBanner
-        @title="Next steps"
-        @type="warning"
-        @message="The CA Chain and Issuing CA values will only be available once. Make sure you copy and save it now."
-      />
-
+      <Hds::Alert @type="inline" @color="highlight" class="has-bottom-margin-s" as |A|>
+        <A.Title>Next steps</A.Title>
+        <A.Description>
+          The CA Chain and Issuing CA values will only be available once. Make sure you copy and save it nowT
```

Comment:
Maybe looks like a typo at the end of this description? T should be a period?

Reply:
good catch!